docs/components/modeler/element-templates/defining-templates.md

```diff
@@ -1,959 +1,105 @@
-
-For example, the following `engines` definition specifies that the template is compatible with Camunda 8.6 or higher.
-
-```json
-{
-  ...,
-  "engines": {
-    "camunda": ">8.5"
-  }
-}
-```
-
-Compatibility is only validated if the platform version is provided by both the template and the modeler. In the example below, the template is compatible with specified versions of both Desktop and Web Modeler, but it requires Camunda version 8.6 or higher for both:
-
-```json
-{
-  ...,
-  "engines": {
-    "camunda": ">8.5",
-    "camundaDesktopModeler": ">=5.30",
-    "camundaWebModeler": "^8.5.5"
-  }
-}
-```
-
-You can also use this feature to explicitly specify a template’s incompatibility with a platform. For instance, the following template is incompatible with all versions of Web Modeler:
-
-```json
-{
-  ...,
-  "engines": {
-    "camundaWebModeler": "0"
-  }
-}
-```
-
-If no `engines` are specified, a template is considered compatible with any execution platform version.
+## Required keys
 
-:::tip
-Review the [upstream documentation](https://github.com/bpmn-io/element-templates/blob/main/docs/LIFE_CYCLE.md#overview) to learn more about template evolution and the life cycle.
-:::
-
-## Supported BPMN types
-
-Currently, element templates may be used on the following BPMN elements:
+- [`$schema : String`](./template-metadata.md#validation-schema): URI pointing towards the [JSON schema](https://json-schema.org/) which defines the structure of the element template `.json` file. Element template schemas are maintained in the [element templates JSON schema](https://github.com/camunda/element-templates-json-schema) repository.
+- [`id : String`](./template-metadata.md#identification-id-and-version): Identifier of the template.
+- [`name : String`](./template-metadata.md#discoverability-name-description-keywords-icon-documentationref-and-category): Name of the template. Shown in the element template selection modal and in the properties panel on the right side of the screen (after applying an element template).
+- [`appliesTo : Array<String>`](./template-metadata.md#supported-bpmn-types-appliesto-and-elementtype): List of BPMN element types the template can be applied to.
+- [`properties : Array<Object>`](./template-properties.md): List of properties that the template applies to the BPMN element. Each property object defines the type of input and how its value is bound to a BPMN or Camunda extension property.
 
-- `bpmn:Activity` (including tasks, service tasks, and others)
-- `bpmn:SequenceFlow` (for maintaining `condition`)
-- `bpmn:Process`
-- `bpmn:Event`
+## Optional keys
 
-## Defining template properties
+- [`version : Integer`](./template-metadata.md#identification-id-and-version): Property to support template versioning and upgrading. If you add a version to a template, it is considered unique based on its ID and version.
```

Comment:
For the future, we could always turn these long lists into tables. 🚀

Reply:
Very good idea. Should we somehow track that as an issue in the docs repo?